sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -3627,6 +3629,24 @@ class DataSourceV2SQLSuiteV1Filter
     }
   }
 
+  test("SPARK-49211: V2 Catalog can also support built-in data sources") {
+    val fullTablename = "testcat3.default.t"
+    withTable(fullTablename) {
+      sql(
+        "CREATE TABLE " + fullTablename + " (name STRING) USING PARQUET LOCATION '/tmp/test_path'")
```

Comment:
This does not test `V1Table`, right? `testcat3` is not a `V2CatalogSupportBuiltinDataSource`

Reply:
oh It's registered at https://github.com/apache/spark/pull/47723/files#diff-0c0de83250682f28a4d129e4b1c0290b2725dcaa36a5eee51ea922f0d53a9a43R44

Let's register here as it's a very special one and very likely will be only used by this test case.

Reply:
sure. moved to only register for the test case.